courses/machine_learning/deepdive2/recommendation_systems/solutions/content_based_by_hand.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<blockquote># each row represents a user's rating for the different movies</blockquote><p>Capitalize each and add period at end of sentence.</p><p> </p><blockquote># features of the movies one-hot encoded</blockquote><p>Change to "Features of the movies are one-hot encoded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/691/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Ack.</p>